plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.cpp

```diff
@@ -1119,6 +1119,7 @@ void TransformationMatrix::initialiseToPrim(bool readFromPrim, Scope* transformN
             MPlug(transformNode->thisMObject(), MPxTransform::shearXZ).setValue(tempShear.y);
             MPlug(transformNode->thisMObject(), MPxTransform::shearYZ).setValue(tempShear.z);
             m_shearTweak[0] = m_shearTweak[1] = m_shearTweak[2] = 0;
+            m_shearFromUsd = tempShear;
```

Comment:
Ideally, this would be on it's own PR or at least commit - both because it's obviously different than the rest of the changes, and because it seems like an important enough bug fix to deserve splitting out.

Having said that - this does seem like a needed fix, and you've already been good enough to split things out into two separate commits, so I'm not going to hold things up over this.